### PR TITLE
added 3 brands

### PR DESCRIPTION
**Pull request by doktorpixel14**
added Penn Station, Monro Muffler Brake and Gordon Food Service

### CODE REVIEW

File: brands/amenity/fast_food.json [hunk 1]
```diff
@@ -1228,6 +1228,23 @@
       "official_name": "Papa Murphy's Take 'N' Bake Pizza",
       "takeaway": "yes"
     }
+  },
+    "amenity/fast_food|Penn Station": {
+    "nocount": true,
+    "countryCodes": ["us"],
+    "match": [
+      "amenity/fast_food|Penn Station",
```
Comment (ENT8R) [inline brands/amenity/fast_food.json]: This can be removed because this is the actual name of the entry...

File: brands/shop/car_repair.json [hunk 2]
```diff
@@ -214,6 +214,17 @@
       "shop": "car_repair"
     }
   },
+  "shop/car_repair|Monro Muffler Brake": {
+    "nocount": true,
+    "countryCodes": ["us"],
```
Comment (ENT8R) [inline brands/shop/car_repair.json]: Maybe this should also have a `match` property for `shop/car_repair|Monro`?

```suggestion
    "countryCodes": ["us"],
    "match": ["shop/car_repair|Monro"],
```

**Comment by doktorpixel14** [comment]
Okay, thanks for your suggestions. I added them to the PR

File: brands/amenity/fast_food.json (hunk 1 above)
Comment (ENT8R) [reply brands/amenity/fast_food.json]: Oh... I'm sorry. I meant that only `amenity/fast_food|Penn Station` should be removed.
`amenity/restaurant|Penn Station` should stay there 😄

**Comment by ENT8R** [comment]
Thank you for this PR 👍 🎉

**Pull request merged**